src/Compilers/CSharp/csc/csc.csproj

```diff
@@ -11,8 +11,9 @@
     <Prefer32Bit>false</Prefer32Bit>
     <LargeAddressAware>true</LargeAddressAware>
     <StartupObject>Microsoft.CodeAnalysis.CSharp.CommandLine.Program</StartupObject>
-    <TargetFramework>net46</TargetFramework>
-    <RuntimeIdentifiers>win7</RuntimeIdentifiers>
+    <TargetFrameworks>$(RoslynPortableTargetFrameworks)</TargetFrameworks>
```

Comment:
we can't use this here. it would cause the compiler to target net461. need to define the new property

Reply:
I'm surprised this got caught by CI! 👍